v1.5/getting-started/kubernetes/installation/hosted/kubeadm/calico.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 data:
   # The location of your etcd cluster.  This uses the Service clusterIP
   # defined below.
-  etcd_endpoints: "http://100.78.232.136:6666"
+  etcd_endpoints: "http://10.96.232.136:6666"
```

Comment:
I'm second guessing myself on whether or not we should change this in v1.5.

This won't work for existing users with older versions of kubeadm.  Definitely think worth doing for v1.6 (which is the latest stable release) and 2.0 (beta).

@gunjan5 WDYT?